CONTRIBUTING.md

```diff
@@ -75,9 +75,13 @@ CREATE EXTENSION pgmq cascade;
 
 ### Installing pg_partman (optional)
 
-If you are working with partitioned queues, you will need to install `pg_partman`.
+If you are working with partitioned queues, you will need to install `pg_partman` version <= 4.7.0
 
 ```bash
+git clone https://github.com/pgpartman/pg_partman.git
+cd pg_partman
+git fetch
+git checkout 4.7.0
 make install-pg-partman
```

Comment:
I dont think we want both the `make` command and the `git` commands here, since these `git` commands are also part of the make command:

https://github.com/tembo-io/pgmq/blob/133356e1823fbdfde4d95b37aa3726901af39061/pgmq-extension/Makefile#L40-L42

Reply:
Thanks @ChuckHend, Taken care